test/test_mkldnn.py

```diff
@@ -381,6 +383,24 @@ def test_zero_(self):
             x2.zero_().to_dense(),
         )
 
+    def _test_imagenet_model(self, model):
+        model = model.train(False).float()
```

Comment:
what's the difference with `train(False)` and `eval()`?

Reply:
exactly the same https://github.com/pytorch/pytorch/blob/6b97279/torch/nn/modules/module.py#L1042